dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -390,6 +420,79 @@ class BGPPeerMgr(Manager):
         return peers
 
 
+def prefix_attr(attr, value):
+    if not value:
+        return None
+    else:
+        try:
+            prefix = netaddr.IPNetwork(str(value))
+        except:
+            return None
+    return str(getattr(prefix, attr))
+
+
+class InterfaceMgr(Manager):
+    def __init__(self, daemon, directory, interface_table = swsscommon.CFG_INTF_TABLE_NAME):
+        super(InterfaceMgr, self).__init__(
+            daemon,
+            directory,
+            [],
+            "CONFIG_DB",
+            interface_table
+        )
+
+    def set_handler(self, key, data):
+        # There are two types of entries for an interface.
+        # One of them is to specify ip and prefix.
+        # In this case, the key contains '|', like "Ethernet0|192.168.0.6/30".
+        # Another one is to specify whether this interface belongs to a vnet
+        # In this case, the key only includes the interface name.
```

Comment:
is this true that vnet interface will only have interface name without ip addresses? @prsunny ?

Reply:
@Pterosaur , can you modify the comment accordingly, current description is not accurate.